cmake/Modules/CocosBuildHelpers.cmake

```diff
@@ -1,28 +1,23 @@
 include(CMakeParseArguments)
 
-# copy libs to static libs folder
-# error function, have bug
-function(cocos_put_static_libs lib_target lib_dir)
-  add_custom_command(TARGET ${lib_target}
-    POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E copy ${lib_dir}/lib${lib_target}.a ${COCOS_PREBUILT_LIBS_PATH}/lib${lib_target}.a
-    COMMENT "${TARGET_NAME} POST_BUILD ..."
-    )
-endfunction()
-
 # lib_name eg cocos2d/cocos2djs
-macro(cocos_find_static_libs lib_name)
-  # only search COCOS_PREBUILT_LIBS_PATH
-  MESSAGE( STATUS "cocos static library path: ${COCOS_PREBUILT_LIBS_PATH}")
-  FIND_LIBRARY(LIB_FOUND ${lib_name} PATHS ${COCOS_PREBUILT_LIBS_PATH} DOC "using cocos static library: lib${lib_name}.a" NO_DEFAULT_PATH NO_CMAKE_FIND_ROOT_PATH)
+function(cocos_find_prebuilt lib_target lib_ret)
```

Comment:
`cocos_find_prebuilt lib_target lib_ret` this function name is strange, what's the meaning? Why not rename to `cocos_find_prebuilt lib`?

Reply:
Oh, `lib_ret` and lib_target  are parameters. But what's the meaning of them?

Reply:
Should I add detail comment for every macro or function parameters? 

In this macro `lib_target ` is a lib name, means which lib do you want to find in prebuilt directory. `lib_ret` is the found lib, it save the absolute path of lib. it's a function `lib_ret` like return value, for cmake's function need to set return value like this `set(${lib_ret} ${tmp_lib} PARENT_SCOPE)`

I think use function is a better choice than macro, it didn't have effect on outer space, we can keep focus on the input parameters and output value.

Reply:
I think it is better to have better names, and add some comments for it, for example

```
# find a prebuilt lib by `name` and save the result in `out`
function(cocos_find_prebuilt_lib_by_name name out)
```

Reply:
agree, I will do some improvements.